examples/freehandRoi/index.html

```diff
@@ -113,7 +114,11 @@ <h4>Edit</h4>
       const data = e.detail;
       const keyCode = data.keyCode;
 
-      if (keyCode === keyCodes.CTRL) {
+      if (keyCode === keyCodes.ESC) {
+        cornerstoneTools.freehand.deactivate(element, 1);
```

Comment:
Wouldn't it be simpler just to call the end drawing function directly rather than deactivating and re-activating?